osbuild-composer.spec

```diff
@@ -303,7 +302,7 @@ Requires:   osbuild >= 93
 Requires:   osbuild-ostree >= 93
 Requires:   osbuild-lvm2 >= 93
 Requires:   osbuild-luks2 >= 93
-Requires:   %{name}-dnf-json = %{version}-%{release}
+Requires:   osbuild-depsolve-dnf = %{version}-%{release}
```

Comment:
This won't work, since the `%{version}-%{release}` is the version and release of `osbuild-composer` package, not of `osbuild`. You need to bump the osbuild version to the one which included this new sub-package.

Reply:
thanks! looked it up and 98 is the first first that includes `osbuild-depsolve-dnf`